src/pages/overlay/components/Select.tsx

```diff
@@ -0,0 +1,43 @@
+import { useCallback, type ChangeEvent } from "react";
+
+interface SelectOption<T extends string = string> {
+  value: T;
+  label: string;
```

Comment:
Can we move this into the sorting util, replacing the current `SortOption` type?

Reply:
I think it'd be fine to move it and use it as the definition for `SortOption`, as it's basically identical (with the added bonus of the generic)

Reply:
I suppose you could import it the other way around too, or maybe best to just keep two definitions?